dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/java/org/apache/dolphinscheduler/api.test/pages/project/WorkFlowDefinitionPage.java

```diff
@@ -0,0 +1,205 @@
+        params.put("searchVal",workFlowName);
+        headers.put(Constants.SESSION_ID_KEY, sessionId);
+        RequestClient requestClient = new RequestClient();
+        ProjectPage project = new ProjectPage();
+        String projectCode = project.getProjectCode(sessionId, projectName);
+        HttpResponse res = requestClient.get("/projects/"+ projectCode + "/process-definition", headers, params);
+
+        for (WorkFlowResponseTotalList workFlowList : JSONUtils.convertValue(res.body().data(), WorkFlowResponseData.class).totalList()) {
+            workFlowCode =  workFlowList.code();
+        }
+
+        return res;
+    }
+
+    public HttpResponse onLineWorkflow(String sessionId, String projectName, String workFlowName){
+        WorkFlowDefinitionPage workflow = new WorkFlowDefinitionPage();
+        workflow.queryWorkflow(sessionId, projectName, workFlowName);
+        Map<String, Object> params = new HashMap<>();
+        Map<String, String> headers = new HashMap<>();
+        params.put("name", workFlowName);
+        params.put("releaseState", "ONLINE");
+        headers.put(Constants.SESSION_ID_KEY, sessionId);
+        RequestClient requestClient = new RequestClient();
+        ProjectPage project = new ProjectPage();
+        String projectCode = project.getProjectCode(sessionId, projectName);
+        HttpResponse res = requestClient.post("/projects/"+projectCode+"/process-definition/"+workFlowCode+"/release", headers, params);
+
+        return res;
+
+    }
+
+    public HttpResponse runWorkflow(String sessionId, String projectName, String workFlowName){
+        WorkFlowDefinitionPage workflow = new WorkFlowDefinitionPage();
+        workflow.queryWorkflow(sessionId, projectName, workFlowName);
+        workflow.onLineWorkflow(sessionId, projectName, workFlowName);
+        Map<String, String> headers = new HashMap<>();
+
+        WorkFlowRunRequestData workFlowRunRequestData = new WorkFlowRunRequestData();
+
+        workFlowRunRequestData.processDefinitionCode(workFlowCode);
+        workFlowRunRequestData.startEndTime("2022-06-25T16:00:00.000Z");
+        workFlowRunRequestData.scheduleTime("2022-06-26 00:00:00,2022-06-26 00:00:00");
+        workFlowRunRequestData.failureStrategy("CONTINUE");
+        workFlowRunRequestData.warningType("NONE");
+        workFlowRunRequestData.warningGroupId("");
+        workFlowRunRequestData.execType("START_PROCESS");
+        workFlowRunRequestData.startNodeList("");
+        workFlowRunRequestData.taskDependType("TASK_POST");
+        workFlowRunRequestData.dependentMode("OFF_MODE");
+        workFlowRunRequestData.runMode("RUN_MODE_SERIAL");
+        workFlowRunRequestData.processInstancePriority("MEDIUM");
+        workFlowRunRequestData.workerGroup("default");
+        workFlowRunRequestData.environmentCode("");
+        workFlowRunRequestData.startParams("");
+        workFlowRunRequestData.expectedParallelismNumber("");
+        workFlowRunRequestData.dryRun(0);
```

Comment:
Same as L75.

Reply:
Thanks for the reminder, it has been fixed